app/src/state/editor/edit.ts

```diff
@@ -109,13 +147,27 @@ function deleteParagraphBreak(state: EditorState, currentIndex: number) {
   if (item.type !== 'paragraph_break') {
     throw new Error('deleteParagraphBreak needs to be called on a paragraph_break');
   }
-  if (currentIndex == firstParagraphBreakIndex(state.document.content)) {
+  if (currentIndex == state.document.content.length - 1) {
     return;
```

Comment:
We need to "protect" the last item in the document, which is the mandatory last paragraph_break of a document.

The change is because with fileformat v2 `paragraph_breaks` we're the "beginning" of a paragraph, so be needed to protect the first item in the document, which set the speaker for the first paragraph